dvc/command/pipeline.py

```diff
@@ -231,7 +232,10 @@ def add_parser(subparsers, parent_parser):
         help="Output DAG as ASCII.",
     )
     pipeline_show_parser.add_argument(
-        "--dot", help="Write DAG in .dot format."
+        "--dot",
```

Comment:
This is breaking backwards-compatibility, but I also think it is worth it, since it will prevent people from accidentally corrupting their dvc files. After this change, people will get "not a dvc file" error, which is not destructive.